lib/cmd/fh3/admin/logs/syslogs.js

```diff
@@ -0,0 +1,71 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var _ = require('underscore');
+
+module.exports = {
+  'desc': i18n._('Search for log entries associated with a request ID'),
+  'examples': [{
+    cmd: 'fhc admin syslogs --requestId="requestid12345" --projects="core,mbaas"',
+    desc: i18n._('Searching for log entries associated with request ID: requestid12345')
```

Comment:
It might be better to use a proper-formed mock request id in the examples instead of one that does not look like one generated by the platform